akka-http/src/main/mima-filters/10.0.6.backwards.excludes

```diff
@@ -0,0 +1,4 @@
+# Provide routes method in HttpApp #953
```

Comment:
I think with the comments in the Mima file we should mostly describe why the exclusions are valid, i.e. in that case because the API in question is marked as `@ApiMayChange`.